roles/build_openstack_packages/templates/projects.ini.j2

```diff
@@ -40,7 +40,8 @@ custom_preprocess={{ cifmw_bop_build_repo_dir }}/DLRN/scripts/set_nvr.sh,{{ cifm
 info_files=osp.yml
 versions_url=https://trunk.rdoproject.org/centos9-{{ cifmw_bop_openstack_release }}/current-podified/versions.csv
 downstream_source_git_branch={{ cifmw_bop_osp_release }}-trunk-patches
-downstream_distro_branch={{ cifmw_bop_osp_release }}{{ '.0' if '.' not in cifmw_bop_osp_release else '' }}-rhel-9-trunk
+# downstream_distro_branch needs to be rhoso not rhos.
+downstream_distro_branch={{ cifmw_bop_osp_release | replace('rhos', 'rhoso') }}{{ '.0' if '.' not in cifmw_bop_osp_release else '' }}-rhel-9-trunk
```

Comment:
{{ '.0' if '.' not in cifmw_bop_osp_release else '' }}

so that is transformign 18 to 18.0  but it it was already 18.0.4 it would not modify it
makes sense

Reply:
the only issue here with replace, if cifmw_bop_osp_release set to rhose, replace will result into bad string. so we change rhos- -> rhoso- ? or just not touch it here and change wherever cifmw_bop_osp_release is set.

Reply:
if we check all the openstack project dist git. They have both branches rhos-18.0-trunk-patches and rhoso-18.0-trunk-patches. But for watcher , it has rhoso-18.0-trunk-patches. That's why I went via this.

Reply:
but this parameter is for distgit branches which ends with rhel-9-trunk, rhos- is legacy, rhoso- is current, watcher must be created recently so have only the current one.
issue is with rhos-> rhoso replacement, "rhos-" -> "rhoso-" i.e including dash should handle the concern i raised above.

Reply:
@karelyatin where do you see that rhos- prefix ix legacy? IIRC it is not.